packages/core/data-transfer/src/strapi/providers/local-destination/strategies/restore/entities.ts

```diff
@@ -31,12 +31,16 @@ const createEntitiesWriteStream = (options: IEntitiesRestoreStreamOptions) => {
         const { create, getDeepPopulateComponentLikeQuery } = query(type);
         const contentType = strapi.getModel(type);
 
+        let cType:
+          | Schema.ContentType
+          | Schema.Component
+          | ((...opts: any[]) => Schema.ContentType | Schema.Component) = contentType;
```

Comment:
This wouldn't be Strapi['getModel']?

Reply:
Nope it is required because the variable is changing inside the loop below and becomes a function sometimes which isn't the simple getModel :)